gulpHelpers.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
     try {
       var absoluteModulePath = path.join(__dirname, MODULE_PATH);
       internalModules = fs.readdirSync(absoluteModulePath)
-        .filter(file => (/^[^\.]+(\.js)?$/).test(file))
+        .filter(file => (/^[^\.]+(\.[jt]s)?$/).test(file))
```

Comment:
There are a couple of other places where we need to consider `ts` and `js` files